speculos/mcu/display.py

```diff
@@ -81,11 +93,33 @@ def draw_point(self, x, y, color):
         self.pixels[(x, y)] = color
 
     def screenshot_update_pixels(self):
+        # Someone tampered our self.pixels content.. update the screenshot to reflect it
+        # NANO only
         self.screenshot.update(self.pixels)
 
     def take_screenshot(self):
         return self.screenshot.get_image()
 
+    def publish_screenshot(self):
+        if self.model == "stax":
+            screen_size, data = self.take_screenshot()
+            self.published_screenshot_value = _screenshot_to_iobytes_value(screen_size, data)
+        else:
+            # Never called
+            pass
+
+    def take_published_screenshot(self):
```

Comment:
I would maybe call this one just `get_screenshot`.

`take` suggests the screenshot is taken when the function is called, which is only true for nano devices

Reply:
`get_public_screenshot`